builtin/logical/aws/backend_test.go

```diff
@@ -948,6 +1147,63 @@ func TestBackend_AssumedRoleWithPolicyARN(t *testing.T) {
 	})
 }
 
+func TestBackend_AssumedRoleWithGroups(t *testing.T) {
```

Comment:
Many of these tests are skipped if `VAULT_ACC` is not on. Could we get a copy/paste of the output from running all the tests in the `builtin/logical/aws` directory with `VAULT_ACC` set to 1?

Reply:
Sure, added to the PR description.